wpilibc/athena/src/AnalogOutput.cpp

```diff
@@ -49,8 +36,13 @@ AnalogOutput::AnalogOutput(uint32_t channel) {
   HalPortHandle port = getPort(m_channel);
   int32_t status = 0;
   m_port = initializeAnalogOutputPort(port, &status);
-  wpi_setErrorWithContext(status, getHALErrorMessage(status));
   freePort(port);
```

Comment:
Since its just an integer value passed back for Athena, it's always copied. It was the old way too. Currently FreePort is a noop, but on future systems it might actually need to free a struct.